cookie-banner-rules-list.json

```diff
@@ -540,34 +537,31 @@
     },
     {
       "click": {
-        "optIn": "button.btn-accept",
-        "optOut": "button.btn-reject",
-        "presence": "div#cookie-disclosure"
+        "optIn": "#cookie-disclosure-accept",
+        "optOut": "#cookie-disclosure-reject",
+        "presence": "#cookie-disclosure"
       },
       "cookies": {},
       "id": "6037802d-9a37-4df2-bf35-9ad60c478725",
       "domains": ["netflix.com"]
     },
     {
       "click": {
-        "optIn": "div[role=dialog] > div > button",
-        "optOut": "div[role=dialog] > div > button + button",
-        "presence": "#scrollview + div"
+        "optIn": "._a9--._a9_0",
+        "optOut": "._a9--._a9_1",
+        "presence": ".x1n2onr6.xzkaem6"
       },
       "cookies": {},
       "id": "e7112653-2ab3-4ddf-b0b1-396052c2f001",
       "domains": ["instagram.com"],
       "filter_expression": "env.appinfo.OS != 'Android'"
     },
     {
-      "click": {},
-      "cookies": {
-        "optOut": [
-          { "name": "ckns_explicit", "value": "2" },
-          { "name": "ckns_policy", "value": "000" },
-          { "name": "ckns_privacy", "value": "july2019" }
-        ]
+      "click": {
+        "presence": "#bbccookies",
+        "optIn": "#bbccookies-continue-button"
```

Comment:
The banner has a "do not agree" button, can we target that too?

![image](https://github.com/mozilla/cookie-banner-rules-list/assets/1077860/1ae2339a-46fd-4fd4-b26a-884c91f37e8f)

Oddly this site has two banners. Since we can't handle both let's target the full screen one instead, which is most disrupting.

Reply:
I don't get the full screen banner.